MobileFilter/sections/specific_web.txt

```diff
@@ -57,6 +57,7 @@ gamesworld.de,fnn.jp##.Ad
 !
 ! SECTION: Regular rules
 !
+||nbcnewyork.app.link/branch-amp-journeys-pre$domain=nbcnewyork.com
```

Comment:
It is used on many sites with different subdomains.
Added as `||app.link/branch-amp-journeys-pre`